lib/cli.js

```diff
@@ -15,9 +15,18 @@ const {
 const { executeThread } = require('./thread');
 const { resultsPath } = require('./shared-config');
 
-function cleanResultsPath() {
-  fs.remove(resultsPath);
-}
+function cleanResultsPath() { 
+  if(!fs.existsSync(resultsPath)) {
+    fs.mkdirSync(resultsPath, { recursive: true }) 
+  } else {
+    fs.readdir(resultsPath, (err, files) => {
+      if (err) console.log(err); 
+        for (const file of files) {
+          fs.unlink(path.join('path here', file), err => { if (err) console.log(err); }); 
```

Comment:
Hi @hiroyone - should the string `"path here"` have been present here?  I'm seeing it surface when I run the tool.

<img width="501" alt="image" src="https://user-images.githubusercontent.com/1315042/208529429-78def59f-69ad-44d4-bd99-358ce9ad8d3a.png">

Reply:
Oh it should be a mistake.
Instead of 'path here', resultsPath seems correct, isn't it?

Reply:
Merged. https://github.com/tnicola/cypress-parallel/pull/149

Reply:
Thank you @hiroyone for updating it so quickly!  I've tested and the error is gone.